### ISSUE TITLE
Compiling directories is broken in 14.x

### ISSUE THREAD

**Issue by stevebest** [jo/couchdb-bootstrap#49]
Running a simple `cdbb http://localhost:5984 ./couchdb` works fine in version 1.14.0, but is completely broken in 14.2.0.

Basically, despite what the docs say, CommonJS `index.js` modules at best make the `cdbb` explode, and at worst - create an unexpected document with a single field called `index` containing its source code, which is totally not what I want.

Project structure:
```
project
  couchdb
    mydb
      index.js
```

Contents of `project/couchdb/mydb/index.js`:
```
module.exports = { _id: "test", foo: "bar" };
```

---

`couchdb-bootstrap@1.14.0` runs as expected, creating a `mydb/test` document.

---

`couchdb-bootstrap@14.2.0` throws an error at `index.js:130`:
```
TypeError: Cannot use 'in' operator to search for '_id' in module.exports = {
    "_id": "test",
    "foo": "bar"
};
```

**Comment by jo** [comment]
Hi @stevebest, thanks for the report!

I have added a testcase and can reproduce the problem.

While I am working on the fix please use couchdb-bootstrap@1.14.0 in the meantime. Sorry for the inconveniences.

**Comment by stevebest** [comment]
Yeah, I guess sometimes jumping over 13 major versions breaks stuff 😜

**Comment by kjames11** [comment]
how can add a new atmosphere

**Comment by edorgeville** [comment]
Hi @jo, the issue is still present in 15.0.0

**Comment by geertmulders** [comment]
We are running into the same problem (same as #54 ). Is there any update on this issue?

**Comment by akdasa** [comment]
Any updates?
